Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Webmaker API
+
+FROM ubuntu:14.10
```

Comment:
We should use an LTS release of Ubuntu instead of the latest version. I'd recommend 14.04.2. We also might want to look at doing our own ubuntu builds, but that can come later